server/services/helpdesk/ZendeskService.ts

```diff
@@ -43,6 +46,9 @@ Email: ${contactHelpdeskForm.email}
             body: messageBody,
           },
           tags: ['SendLegalMail', externalUser ? 'slm_legal_sender' : 'slm_mailroom'],
+          requester: {
+            email: `${contactHelpdeskForm.email}`,
+          },
```

Comment:
So we're going with the solution where every helpdesk request creates a new user on Zendesk? Interesting.